data/json/monsters/mammal.json

```diff
@@ -314,7 +314,7 @@
     "melee_cut": 6,
     "dodge": 2,
     "armor_bash": 2,
-    "starting_ammo": { "milk": 4 },
+    "starting_ammo": { "milk": 10 },
     "path_settings": { "max_dist": 10 },
```

Comment:
What happened to milk yields?

Reply:
> What happened to milk yields?

Changes from #24075 gone missing when #23930 was merged and milkable animals were moved from `monsters.json` to `mammal.json`.